src/core/lwt.ml

```diff
@@ -1,9 +1,9 @@
-(* Lightweight thread library for OCaml
+(* OCaml promise library
```

Comment:
I agree, but saving for a future issue. I want to also start a history of Lwt question thread on Discuss, to find out what people know. I heard it said that Lwt started in 1995 (??!?!?) as a module in some bigger project, perhaps even before Ocsigen.